blogSignUp.php

```diff
@@ -0,0 +1,53 @@
+<?php
+include ('include/include_all.php');
+echo "<link rel='stylesheet' href='maindesign.css' />";
+
+$Errorz = array();
+
+function ValidateSignUp(){
```

Comment:
I won't comment on this in every place, but functions should go in /include probably. In this case, I don't think this needs to be in a function at all. For the most part, you only need a function if you're going to call it multiple times. For specialized things like validating a specific form, you're normally only going to have it in one place.